index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>CountDown</title>
-    <script src="script.js" defer></script>
```

Comment:
I moved the <script> line to the end after the body, otherwise the text of the script showed up after closing the Activities page